rootfs/scheduler/__init__.py

```diff
@@ -993,6 +993,10 @@ def _wait_until_pods_are_ready(self, namespace, container, labels, desired):  #
             logger.info('timed out ({}s) waiting for pods to come up in namespace {}'.format(timeout, namespace))  # noqa
 
         logger.info("{} out of {} pods in namespace {} are in service".format(count, desired, namespace))  # noqa
+        if count != desired:
+            # raising to allow operations to rollback
+            raise KubeException('Not enough pods in namespace {} came into service. '
+                                '{} out of {}'.format(namespace, count, desired))
```

Comment:
yes i know that it has been hardcoded since its inception but now that we are starting to fail the operation based on the timeout i think we should start allowing the user to configure it failing which it might become problem for them....since HEALTHCHECK_INITIAL_DELAY is not mandatory to be set for every pod we shouldn't depend on it